### PR TITLE
67 functionality total sum button

### LINKED ISSUE #67: Functionality - Total Sum button
Add functionality to the total sum button by using the same strategy used on the rules section

### PR DESCRIPTION
**Pull request by met4000**
Closes #67 

(Re)-implementation of the total sum constraint; provides feedback to participant users of the current and required total sum, and prevents submission (by disabling the Qualtrics 'next' button) unless the total sum is correct.

### CODE REVIEW

**Comment by met4000** [comment]
Note: if multiple graph questions are put in the same block in the survey (the same page of a survey), no communication between the questions is done; when any one of the graphs become valid, the 'next' button will be enabled, and when any one of the graphs become invalid, the 'next' button will be disabled.

**Review by LKBEAST — APPROVED** [review]
Main concern is with all the changes to injection.mjs, but nothing appears to be wrong lets hope it all works out

File: components/participant_interface_main/participant_interface.css [hunk 1]
```diff
@@ -138,8 +138,36 @@ select {
   border-radius: 20px;
   border: solid 3px rgb(0, 114, 208);
   background: white;
+}
+
+
+/* total sum display */
+/* ! PLACEHOLDER ! */
```
Comment (LKBEAST) [inline components/participant_interface_main/participant_interface.css:145]: do we get rid of this?

Comment (met4000) [reply components/participant_interface_main/participant_interface.css:145]: might be changed later

**Review by LKBEAST — APPROVED** [review]
LGTM

**Pull request merged**